### DISCUSSION TITLE
Ingesting local Suricata logs with the agent

### DISCUSSION THREAD

**Discussion by ihuxn4hi** [Security-Onion-Solutions/securityonion#11377] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

32

### Storage for /

256

### Storage for /nsm

200

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Is it possible to configure the agent to ingest Suricata's /var/log/suricata/eve.json file that is written locally on an endpoint?

In the stock Elastic, you can just define the path:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/123765379/d8eabb53-a879-4415-ab86-2e6220e38bde)

But that doesn't seem to be possible with the Security Onion build.

I'm in AWS and setting up an enterprise deployment with mirrored interfaces sending data to a Forward Node is prohibitively expensive. Obviously the functionality exists since that's the out of the box architecture with stock Elastic so it would be nice if you at least had the option to use it like that.

Thanks!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by ihuxn4hi** [comment]
Hello? Is this thing on?

**Comment by dougburks** [comment]
You may be able to make this work by copying and modifying the integration but this is untested and unsupported since our official recommendation is to deploy a Security Onion forward node.
